Source/WebInspectorUI/UserInterface/Models/ScreenshotsInstrument.js

```diff
@@ -36,6 +36,10 @@ WI.ScreenshotsInstrument = class ScreenshotsInstrument extends WI.Instrument
 
     static supported()
     {
+        // FIXME: <webkit.org/b/251113> Screenshots timeline does not capture screenshots for changes in WebKitLegacy.
+        if (WI.sharedApp.debuggableType === WI.DebuggableType.Page)
```

Comment:
Will this work for iOS 13.0 and earlier where we have to wait for a `Target.exists` command to fail before "realizing" that we're dealing with WKL?

NIT: I wonder if we should create a `WI.AppControllerBase.prototype.get isWebKitLegacy` (similar to the `WI.AppControllerBase.prototype.get isWebDebuggable` we already have)

Reply:
Not sure about pre-iOS 13. I'll take a look. But the check below this one will fail pre-iOS 16.0. But if we abstract this logic out I'll have to double-check pre-iOS 13 compatibility.

Reply:
oh lol good point i forgot about the other check 🤣